src/client/ui/select-element.js

```diff
@@ -36,6 +37,23 @@ function onDocumentMouseDown (e) {
         collapseOptionList();
 }
 
+function onWindowClick (e, dispatched, preventDefault) {
+    const target   = e.target || e.srcElement;
+    const optionIndex = arrayUtils.indexOf(options, target);
+
+    if (optionIndex < 0)
+        return;
+
+    preventDefault();
+
+    const isDisabled = target.className.indexOf(DISABLED_CLASS) > -1;
+
+    if (isDisabled && browserUtils.isWebKit)
+        return;
```

Comment:
Later we return `false` in this case

Reply:
We returned false in onclick handler of shadow option element. Now we handle click on window so we dont need it at all. Events will be prevented in any cases